lxd/auth/driver_common.go

```diff
@@ -5,15 +5,15 @@ import (
 )
 
 type commonAuthorizer struct {
-	name            string
-	config          map[string]any
-	logger          logger.Logger
-	projectsGetFunc func() (map[int64]string, error)
+	driver string
+	logger logger.Logger
 }
 
-func (c *commonAuthorizer) init(name string, config map[string]any, l logger.Logger, projectsGetFunc func() (map[int64]string, error)) {
-	c.name = name
-	c.config = config
+func (c *commonAuthorizer) init(driver string, l logger.Logger) {
+	c.driver = driver
 	c.logger = l
```

Comment:
Shall we wrap the logger with the driver name, ala

```
func (n *common) init() {
n.logger = logger.AddContext(logger.Ctx{"project": projectName, "driver": netInfo.Type, "network": netInfo.Name})
}
```

Reply:
Done.